qa/tasks/cephfs/test_volumes.py

```diff
@@ -800,6 +800,98 @@ def test_subvolume_group_info(self):
         # remove group
         self._fs_cmd("subvolumegroup", "rm", self.volname, group)
 
+    def test_subvolume_group_quota_subvolume_removal(self):
+        """
+        Tests subvolume removal if it's group quota is set.
+        Validates it doesn't hit https://tracker.ceph.com/issues/16884 as subvolume deletion
+        used to rename the subvolume directory to trash directory which resides outside the
```

Comment:
Does this still hold true?  AIUI, this issue is not hit by disabling quota for the mgr's libcephfs client, and not by changing the path of the trash directory. Don't we continue to rename the subvolume directory to trash directory that resides outside the subvolume group?

Same comment for tests below.

Reply:
Yes, you are right. The comment is no longer valid, I will remove the comment and keep the testcase.